src/main/java/skajihara/projectX/MainContents/Home/exception/AccountExceptionControllerAdvice.java

```diff
@@ -0,0 +1,15 @@
+package skajihara.projectX.MainContents.Home.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class AccountExceptionControllerAdvice {
+
+    @ExceptionHandler(AccountException.class)
+    public ResponseEntity<String> handleAccountException(AccountException ex) {
```

Comment:
ここを `AccountException` ではなく `NotFoundException` を受けれるようにしたら良いかも？
```suggestion
    @ExceptionHandler(NotFoundException.class)
    public ResponseEntity<String> handleNotFoundException(NotFoundException ex) {
```
実際に `HttpStatus.NOT_FOUND` を返却してますから、そのほうが直感的だと考えます。

Reply:
`NotFoundException` を受けれるように修正しました。